### PR TITLE
Add arm ubuntu build

### PR DESCRIPTION
**Pull request by rishflab**
proof that it works: https://github.com/rishflab/rendezvous-server/actions/runs/992703475

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -43,6 +43,7 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt-get install gcc-arm-linux-gnueabihf
+          rustup target add armv7-unknown-linux-gnueabihf
```
Comment (thomaseizinger) [inline .github/workflows/ci.yml:46]: If you add a rust-toolchain.toml to the repo like we do in xmr-btc, then you don't even have to add this line! :)

File: .github/workflows/build-release-binaries.yml [hunk 2]
```diff
@@ -14,6 +14,10 @@ jobs:
             target: x86_64-unknown-linux-gnu
             os: ubuntu-latest
             archive_ext: tar
+          - bin: rendezvous_server
```
Comment (thomaseizinger) [inline .github/workflows/build-release-binaries.yml:17]: Given that we only have on binary in here, we shouldn't need that :)

**Review by thomaseizinger — APPROVED** [review]
Two things that could be optimized. Good to go this way though!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (rishflab) [reply .github/workflows/ci.yml]: https://github.com/comit-network/rendezvous-server/runs/2969565537
doesn't work

Comment (thomaseizinger) [reply .github/workflows/ci.yml:46]: I don't see a `rust-toolchain.toml` file [in the repository at the commit](https://github.com/comit-network/rendezvous-server/tree/7c5250d4589cca73a293b9fa11e8c8fe7a9b6fd4) that the workflow you linked to ran at.

Comment (rishflab) [reply .github/workflows/ci.yml:46]: I dropped the commit and force pushed. I will try again and leave the commit up to make sure

**Pull request merged**